EIPS/eip-xxxx.md

```diff
@@ -0,0 +1,42 @@
+---
+eip: XXXX
+title: Increase Maximum Contract Size to 48KB and initcode Size to 72KB
+status: Draft
+author: Giulio Rebuffo (@Giulio2002), Ben Adams (@badams)
+discussions-to: https://ethereum-magicians.org/t/increase-maximum-contract-size-to-48kb/24509
+type: Core
+category: Core
+created: 2025-06-09
+requires: 1
```

Comment:
Requires 170 otherwise there is no limit :smile: :+1: And 3860 for initcode limit